src/cursor/cur_hs.c

```diff
@@ -12,12 +12,16 @@
 static int __curhs_prev_visible(WT_SESSION_IMPL *, WT_CURSOR_HS *);
 static int __curhs_next_visible(WT_SESSION_IMPL *, WT_CURSOR_HS *);
 static int __curhs_search_near_helper(WT_SESSION_IMPL *, WT_CURSOR *, bool);
+static int __file_cursor_next(WT_SESSION_IMPL *, WT_CURSOR *);
```

Comment:
I am not sure about using `__file` prefix here. We have following a convention about using module name as prefix.